src/components/pages/search.js

```diff
@@ -28,28 +27,18 @@ function Search({ initialState, PageTemplate }) {
     () => docs
       .map(doc => ({
         ...doc,
+        tags: [...new Set(doc.tags)], // Some docs have duplicate tags we don't want to render
```

Comment:
Why keep the code when we have this as a next thing?

Reply:
We don't have it yet, so currently we have duplicate key errors